models/l3afd.go

```diff
@@ -72,3 +72,17 @@ type BPFPrograms struct {
 	TCIngress  []*BPFProgram `json:"tc_ingress"`  // list of tc ingress bpf programs
 	TCEgress   []*BPFProgram `json:"tc_egress"`   // list of tc egress bpf programs
 }
+
+// L3afBPFProgramNames defines names of Bpf programs on interface
+type L3afBPFProgramNames struct {
+	HostName        string           `json:"host_name"`    // Host name or pod name
+	Iface           string           `json:"iface"`        // Interface name
+	BpfProgramNames *BPFProgramNames `json:"bpf_programs"` // List of bpf program names to remove
+}
+
+// BPFProgramNames defines names eBPF of programs on node
```

Comment:
```suggestion
// BPFProgramNames defines names of eBPF programs on node
```
(I think this was my error in my earlier comment)